charts/go-boilerplate-ddd/templates/hpa.yaml

```
@@ -0,0 +1,33 @@
+  namespace: {{ include "global.namespace" . }}
+  labels:
+    {{- include "boilerplate.labels" (dict "context" . "component" .Values.boilerplate.name "name" .Values.boilerplate.name) | nindent 4 }}
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: {{ include "boilerplate.fullname" . }}
+  minReplicas: {{ .Values.boilerplate.autoscaling.minReplicas }}
+  maxReplicas: {{ .Values.boilerplate.autoscaling.maxReplicas }}
+  metrics:
+    {{- if .Values.boilerplate.autoscaling.targetCPUUtilizationPercentage }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)